src/pages/map.html

```diff
@@ -1,6 +1,36 @@
 <div class="">
     <div class="h-28 -mt-28" id="curve-to-start-Qualitat-section" style="background: url(./assets/BGs/qualitat.svg); background-position: top; background-size: cover;"></div>
-    <div class="text-white bg-[#001342]" id="rwanda-map-section-description">Rwanda map section description</div>
+    <div class="text-white bg-[#001342] flex flex-col gap-5 px-24">
+      <div class="font-bold text-3xl w-full font-raleway">Ruanda-ein Land der <span class="text-green-primary">Sub-Sahara</span> mit viel potential</div>
+      <div class="max-w-2xl">
+        <p class="pb-5 font-montserrat">
+          kurze Einführung über 4-5 Zeilen At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est L  vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est L
+          </p>
+      </div>
+      <div class="flex mb-24">
+        <div class="ml-auto relative items-center pl-40">
+          <img src="../../assets/images/Continent.svg" class="w-5/6 relative" alt="">
+          <div class="absolute top-[385px] left-[637px] flex gap-3">
+            <img src="../../assets/images/image.svg"  alt="">
+          <div class=" flex flex-col">
+            <h2 class="font-bold w-full relative font-montserrat">Ruanda<div class="h-1 bg-green-primary-trans w-16 absolute top-[18px] -left-0 -rotate-3"></div></h2>
+            <p class="font-montserrat text-xs font-light">Hauptstadt:Kigali</p>
+          </div>
+          </div>
+          
+        </div>
+        <div class=" absolute bg-gradient-to-r from-green-primary-trans/50 to-transparent to-[60%]  mt-[250px] w-3/4 p-16 px-32 flex flex-col gap-8">
```

Comment:
Here too you need to remove this arbitrary value. Use '3/5' instead of '[60%]'

Reply:
3/5 throws an error it only accepts percentages.

Reply:
okay